attributes/default.rb

```diff
@@ -33,15 +33,20 @@
 
 # Install related attributes
 default['redisio']['safe_install'] = true
+default['redisio']['package_install'] = false
+default['redisio']['package_name'] = 'redis'
 default['redisio']['bypass_setup'] = false
 
 # Tarball and download related defaults
 default['redisio']['mirror'] = "http://download.redis.io/releases/"
 default['redisio']['base_name'] = 'redis-'
 default['redisio']['artifact_type'] = 'tar.gz'
-default['redisio']['version'] = '2.8.17'
 default['redisio']['base_piddir'] = '/var/run/redis'
 
+# Version
+default['redisio']['version'] = '2.8.17' # specify a version
+#default['redisio']['version'] = nil # latest version (only for package install)
```

Comment:
suggestion: package version should be nil by default (rely on mainstream repo version)

```ruby
if node['redisio']['package_install']
  default['redisio']['version'] = nil
else
  default['redisio']['version'] = '2.8.17'
end
```